lib/action_kit/client.rb

```diff
@@ -4,8 +4,8 @@ module Client
     def client(verb, path, params)
       Typhoeus::Request.send(
         verb,
-        "https://act.sumofus.org/rest/v1/#{path}/",
-        { userpwd: "#{ENV['AK_USERNAME']}:#{ENV['AK_PASSWORD']}" }.merge(params)
+        "#{Settings.ak_processor_url}/#{path}/",
```

Comment:
It was already an environment variable, we were just ignoring it in deference to a magic string before.